### ISSUE TITLE
Question on the "File checksum" intent

### ISSUE THREAD

**Issue by paulmillar** [spdx/spdx-spec#623]
Section "8.4.2 Intent" states:
> Here, by providing a unique identifier of each file, confusion over which version/modification of a specific file should be eliminated.

While I appreciate this use (although, a checksum value is not guaranteed to be unique) there is another intent for publishing checksum information.

The other intent is that a checksum (algorithm, value) pair allows someone (with access to the file's data) to verify the integrity of that data.

Perhaps section 8.4.2 could be updated to include this other intention?

**Comment by goneall** [comment]
This may be a duplicate of issue #500 

I would suggest we update this in the 2.3 release

@paulmillar - we're going to create a developement/2.3 branch - once that is done, would you be interested in providing a pull request to update the language?

**Comment by paulmillar** [comment]
Yup, sorry, this looks like a dup of #500.

Yes, I'd be happy to create a pull request against development/2.3 branch.  Just give me a shout when you're ready.

**Comment by goneall** [comment]
@kestewart @tsteenbe Can you ping @paulmillar once the 2.3 fork is made?

**Comment by kestewart** [comment]
@paulmillar - we have a 2.3 branch now,  please link the PR here so we can track effectively.
